frontend/src/stimulus/controllers/op-application.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { ControllerConstructor } from '@hotwired/stimulus/dist/types/core/controller';
+import { ApplicationController } from 'stimulus-use';
+
+export class OpApplicationController extends ApplicationController {
+  declare readonly dynamicTargets:HTMLElement[];
+
+  static targets = ['dynamic'];
+
+  private loaded = new Set<string>();
+
+  public initialize():void {
+    this.bootstrapDynamicControllers();
+  }
```

Comment:
Wouldn't it be more to the resilient if a `dynamicTargetConnected` were to be used? The `initialize` callback is only called once when the controller is initialized. But with this controller residing at the top level of the DOM structure, the DOM governed by it might change later on by server responses. Then it would be good for the controller to also pick up the controllers to be loaded for the added DOM. 

For now, this definitely not a problem but it would be interesting to see if the approach chosen here will also work with dynamically added DOM.

Reply:
Indeed that would be better, I changed it to that approach